src/altair/block_processing.rs

```diff
@@ -315,9 +315,9 @@ pub fn process_sync_aggregate<
     let proposer_reward =
         participant_reward * PROPOSER_WEIGHT / (WEIGHT_DENOMINATOR - PROPOSER_WEIGHT);
 
+    let cloned_validators = state.validators.clone();
```

Comment:
same -- why do we need to change this in the first place?

Reply:
This is due to the iteration and collecting into the Hashbrown's Hashset crate